### ISSUE TITLE
tenant id format

### ISSUE THREAD

**Issue by mwear** [w3c/trace-context#316]
The spec indicates that tenant id must begin with `lcalpha`.  I think it will be common for tenant ids to be numeric and for vendors to use numeric ids or a hash of the id. In either case, the resulting id may begin with a number or a string. While this can be worked around, it would be easier for vendors if the tenant id could begin with `lcalpha` or a `DIGIT`.

```
key = lcalpha 0*240( lcalpha / DIGIT / "_" / "-"/ "*" / "/" ) "@" lcalpha 0*13( lcalpha / DIGIT / "_" / "-"/ "*" / "/" )
```

Is there a reason for restricting identifiers with a tenant_id to starting with `lcalpha`only and would allowing them to start with `lcalpha` or `DIGIT` be possible?

**Comment by danielkhan** [comment]
@SergeyKanzhelev - do you have any objections against removing the preceeding lcalpha?

**Comment by mtwo** [comment]
SGTM as long as this isn't considered a breaking change

**Comment by mwear** [comment]
I believe @plehegar said this would be ok because we relaxing the restriction; if went the other direction it would be a problem.

**Comment by mwear** [comment]
Closing, this was addressed in #322.
